app/styles/global/_form.scss

```diff
@@ -0,0 +1,120 @@
+.tomster-form {
```

Comment:
I've literally copy-and-pasted the existing styles from the current site on dumped them in here. There's been no attempt to try and optimize these styles.

Reply:
that's fine- our goal is to convert then iterate to make better